client/src/components/Dashboard/index.tsx

```diff
@@ -107,7 +107,7 @@ export default class Dashboard extends React.Component<IDashboardProps, IDashboa
 
       // For each column, create a layout according to number of columns
       let layouts = ElementConnector.loadLayoutFromDashboard(dashboard, dashboard);
-      layouts = _.extend(layouts, dashboard.config.layout.layouts || {});
+      layouts = _.extend(layouts, dashboard.layouts || {});
 
       this.layouts = layouts;
```

Comment:
@morsh 
`let layouts` is redundant. I advise to load it directly to `this.layouts` instead.